Orange/widgets/evaluate/owpredictions.py

```diff
@@ -136,7 +136,7 @@ def __init__(self):
             gui.checkBox(
                 predopts, self, "show_probability_errors",
                 "Show classification errors",
-                tooltip="Show 1 - probability assigned to the correct class",
+                tooltip="Show probability assigned to the correct class",
```

Comment:
Looked correct to me the way it was before?
The error is the sum of all incorrect probs i.e. 1 - prob of correct.
The wording however could be improved.